client/my-sites/plans/jetpack-plans/product-store/see-all-features.tsx

```diff
@@ -0,0 +1,21 @@
+import { Button } from '@automattic/components';
+import { useTranslate } from 'i18n-calypso';
+import { recordTracksEvent } from 'calypso/lib/analytics/tracks';
+import { PLAN_COMPARISON_PAGE } from '../constants';
+
+export const SeeAllFeatures: React.FC = () => {
+	const translate = useTranslate();
+
+	return (
+		<div className="jetpack-product-store__see-all-features">
+			<Button
+				onClick={ () => recordTracksEvent( 'calypso_product_seel_all_features_click' ) }
```

Comment:
There's a small typo here, should be `calypso_product_see_all_features_click`

Reply:
thanks for spotting this @andrii-lysenko .. 👍